Fixes #1161

mtp1376 opened the pull request:
Adds the `TELEPRESENCE_TEST_HOSTNAME` environment variable so that `test_hostname` can be changed as needed.
Also changes the default value of `test_hostname` to `kubernetes.default`.

ark3 commented:
Thanks for the PR! I don't want to add optional environment variables (or other configurability) without a cohesive plan for documentation and testing (#985). Can you scale this back to just a hard-coded change for the hostname?

mtp1376 commented:
@ark3 I changed the hard-coded hostname and reverted the environment variable input. About the environment variables, I'll be glad if I can be a part of the planning for the cohesive documentation and testing of them.
